lumeer-core/src/main/java/io/lumeer/core/PermissionsChecker.java

```diff
@@ -75,13 +77,20 @@ private boolean hasRole(Resource resource, Role role) {
     */
    public Set<Role> getActualRoles(Resource resource) {
       String user = authenticatedUser.getUserEmail();
-      Set<String> groups = userCache.getUser(user).getGroups();
+      Set<String> groups = getUserGroups(user, resource);
 
       Set<Role> actualRoles = getActualUserRoles(resource.getPermissions().getUserPermissions(), user);
       actualRoles.addAll(getActualGroupRoles(resource.getPermissions().getGroupPermissions(), groups));
       return actualRoles;
    }
 
+   private Set<String> getUserGroups(String user, Resource resource) {
+      if (resource instanceof Organization) {
+         return Collections.emptySet();
+      }
```

Comment:
Why is this condition here?

Reply:
Because checking groups when dealing with organization rights does not make sense. Groups are stored per organization but you would need something above it to be able to check these access rights.